docs/source/plugins/plugin_visualization.rst

```diff
@@ -158,11 +158,13 @@ Those are the scripts that are used for the command line interface.
     from asreviewcontrib.visualization.plot import Plot
 
     with Plot.from_paths(["PATH_1", "PATH_2"]) as plot:
-        inc_plot = plot.new("inclusion")
+        inc_plot = plot.new("inclusion", result_format="number")
         inc_plot.set_grid()
         inc_plot.set_xlim(0, 30)
         inc_plot.set_ylim(0, 101)
         inc_plot.set_legend()
+        inc_plot.add_wss('filename.h5', value=100)
```

Comment:
The file name is slightly confusing. Do you know what it is? Is it a position argument or keyword?